README.md

```diff
@@ -62,45 +45,64 @@ CREATE EXTENSION postgis_topology;
 $ rails db:setup
 ```
 
-#### Tests
-
-You can run the full test suite using following command:
-
-```sh
-$ rspec
-```
-
-Individual system specs with opening Chrome browser can be run using the following command:
-
-```sh
-$ JS_DRIVER=selenium_chrome rspec spec/system/log_in_spec.rb
-```
-
 #### Start the server:
 
 ```sh
 $ rails server
 ```
 
-#### Because of the subdomain being enforced, your options to test subdomains include using bops-care.link (needed to test [BOPS-applicants](https://github.com/unboxed/bops-applicants)) or, if you're just testing BOPS, Chrome, Firefox and Brave currently support subdomains too. This means your app will be available on:
+## Subdomains
+
+Because of the local authority being inferred on the request's
+subdomain, your options to get work locally include using Docker or
+using the `bops-care.link` domain which points back to your localhost:
 
 ```
 http://southwark.bops-care.link:3000/
-
 http://lambeth.bops-care.link:3000/
-
 http://buckinghamshire.bops-care.link:3000/
+```
 
-or
+Otherwise you can use localhost though you'll have to double the
+subdomain since `localhost` is missing the second componetn found in
+`normal-domains.com`.
 
+```
 http://southwark.southwark.localhost:3000/
-
 http://lambeth.lambeth.localhost:3000/
-
 http://buckinghamshire.buckinghamshire.localhost:3000/
+```
+
+## BOPS applicants
 
+BOPS allows planning officers to request changes to an application;
+these requests are presented to applicants through a separate app
+called
+[BOPS-applicants](https://github.com/unboxed/bops-applicants). Applicants
+receive an email containing a special URL that will be opened in BOPS
+applicants and contain the right paramaters for it to query back at
+BOPS.
+
+If you're using Docker, `bops-applicants` is already part of the
+Compose group and should be running on port 3001. If you're not,
+you'll have to clone/setup/boot the app manually and points BOPS to it
+through the environment variable `APPLICANTS_APP_HOST`.
```

Comment:
which can be set with ```export APPLICANTS_APP_HOST=bops-care.link:3001``` - otherwise people won't know what to set APPLICANTS_APP_HOST to